src/main/java/com/uvocab/api/controller/UserController.java

```diff
@@ -7,7 +7,7 @@
 import uvocab.protobuf.v1.User;
 
 @RestController
-@RequestMapping("/v1/user")
+@RequestMapping("/v1/register")
```

Comment:
@Danilosimoes we don't need change the route name,  the http post verb says for us that it save/register a user.

[here](https://restfulapi.net/http-methods/#post) you can see some example